### DISCUSSION TITLE
Voto non visibile alla famiglia in media

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#583] [Idee e proposte]
Salve sarebbe possibile mettere il voto che fa media quando io metto la spunta su la voce non visibile alla famiglia?

**Comment by trinko** [comment]
No, questo proprio no. Sarebbe molto grave: stai dicendo che un docente mette una valutazione, che sarà usata per la media da cui scaturisce il voto in pagella, ma nasconde a genitori/alunni questa valutazione...
Oltre ad essere fuori da ogni principio di trasparenza e da qualsiasi patto educativo, sarebbe anche oggetto di sicuro ricorso al TAR in caso di bocciatura finale dell'alunno.

**Comment by Davide031** [comment]
Intendevo che un docente mette il voto (non visibile) ma poi o lo toglie o lo mette visibile.

**Comment by trinko** [comment]
Questo si può già fare. Ma nel periodo in cui il voto non è visibile non può essere considerato per le medie automatiche.
